setup.py

```diff
@@ -1,4 +1,5 @@
-from distutils.core import setup
+#from distutils.core import setup
```

Comment:
don't comment, just delete

Reply:
Sure, if you're okay with the change I'll commit to it.